package.json

```diff
@@ -20,6 +20,8 @@
     "date-fns": "^2.0.0-beta.5",
     "file-saver": "^2.0.5",
     "firebase": "^7.24.0",
+    "firebase-admin": "^11.5.0",
+    "firebase-functions": "^4.2.1",
```

Comment:
I don't know if these two firebase packages should be dev dependencies?

Reply:
I think these might have been needed in the project in order to run the emulator?